### PR TITLE
Add delay param to docs

### PR DESCRIPTION
**Pull request by marinhero**
- Add delay param and example to docs

Trello: https://trello.com/c/M52GNJmq/2507-4123-spring-implement-time-delay-for-email-surveys-before-sending-through-api-in-days-3

### CODE REVIEW

File: source/includes/_email_survey.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ survey_settings | Hash | See **survey_settings** parameters below
 subject | String | Override default subject line (standard NPS/CES/CSAT question used by default). Can handle placeholder values surrounded by {{my_value}}
 intro | String | Intro message for your email. Displayed between logo and question. Default: none. Can handle placeholder values surrounded by {{my_value}}
 context | Hash Array | Array of hashes containing the values to be replaced in subject and/or intro
+delay | Integer | Value (in days)  that specifies how many days will the survey wait until being fired
```
Comment (diegoserranoa) [inline source/includes/_email_survey.md]: There's kind of like an extra space there after the parenthesis. Can you verify with Ray or Dan the language of the description?

**Review by diegoserranoa — COMMENTED** [review]
Looks good!

**Review by diegoserranoa — APPROVED** [review]
LGTM

File: source/includes/_email_survey.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ survey_settings | Hash | See **survey_settings** parameters below
 subject | String | Override default subject line (standard NPS/CES/CSAT question used by default). Can handle placeholder values surrounded by {{my_value}}
 intro | String | Intro message for your email. Displayed between logo and question. Default: none. Can handle placeholder values surrounded by {{my_value}}
 context | Hash Array | Array of hashes containing the values to be replaced in subject and/or intro
+delay | Integer | Days the survey will wait until being sent
```
Comment (hachibu) [inline source/includes/_email_survey.md]: Day we will wait before sending the survey

File: source/includes/_email_survey.md [hunk 3]
```diff
@@ -56,6 +57,16 @@ curl "https://api.wootric.com/v1/email_survey" \
   -d "context[][color]=red, blue"
 ```
 
+```sh
+# Example of an eight day delay
```
Comment (hachibu) [inline source/includes/_email_survey.md]: eight-day should be hyphenated

Comment (hachibu) [reply source/includes/_email_survey.md]: https://www.vappingo.com/word-blog/when-to-hyphenate-numbers/

Comment (hachibu) [inline source/includes/_email_survey.md]: I would write the number 8 instead of the word eight.

**Comment by marinhero** [comment]
Done!

**Review by hachibu — APPROVED** [review]
Ship it

**Pull request merged**